integration/test/ParseObjectTest.js

```diff
@@ -2063,18 +2063,32 @@ describe('Parse Object', () => {
   it('allowCustomObjectId', async () => {
     await reconfigureServer({ allowCustomObjectId: true });
     Parse.allowCustomObjectId = true;
+
+    // Try to save without passing objectId
```

Comment:
If a test title doesn't tell exactly what it tests and it needs comments to explain, that's usually a sign that it should be split into individual tests. Combining multiple test scenarios in a single test makes the tests less robust and more difficult to change in the future.

You could add a new `describe` layer for `allowCustomObjectId` and the add tests like:
```
describe('allowCustomObjectId', () => {
  it('fails to save with empty objectId', async () => {
```

Could you split this into individual tests?

Reply:
It was like that previously.

Reply:
Hi @mtrezza! I have split the tests for allowCustomObjectId. Please review again.